docs/httplug/plugins/authentication.md

```diff
@@ -1,3 +1,16 @@
 # Authentication Plugin
 
-TODO: explain the authentication plugin
+This plugin uses the [authentication component](/components/authentication) to authenticate all requests sent through
```

Comment:
s/authenticate all requests/authenticate requests/

we have the filter thingy that allows to only authenticate some requests rather than all.